include/my_sys.h

```diff
@@ -185,8 +185,10 @@ extern BOOL my_obtain_privilege(LPCSTR lpPrivilege);
 void my_init_atomic_write(void);
 #ifdef __linux__
 my_bool my_test_if_atomic_write(File handle, int pagesize);
+my_bool my_test_if_disable_punch_hole(File handle);
 #else
-#define my_test_if_atomic_write(A, B) 0
+#define my_test_if_atomic_write(A, B)       0
+#define my_test_if_disable_punch_hole(A, B) 0
```

Comment:
wrong number of parameters in this macro. It will fail to compile everywhere except Linux.